src/containers/Questions/index.js

```diff
@@ -1,8 +1,122 @@
 import React from 'react';
-import UnderConstruction from '../UnderConstruction';
+import propTypes from 'prop-types';
+import { message } from 'antd';
+import Question from './questions';
 
-const Questions = props => {
-  return <UnderConstruction {...props} />;
+import schema from './questionValidation';
+import entryData from './data';
+
+class Questions extends React.Component {
+  state = {
+    current: 0,
+    title: '',
+    content: '',
+    errors: {},
+    journals: [],
+  };
+
+  handleConfirm = e => {
+    message.warning("You didn't make an entry today");
+    const { history } = this.props;
+    history.push('/home');
+  };
+
+  handleChange = ({ target: { value, name } }) => {
+    this.setState({ [name]: value, errors: {} });
+  };
+
+  handleNext = async () => {
+    const { current, title, content, journals } = this.state;
+    try {
+      await schema.validate({ title, content }, { abortEarly: false });
+      journals.push({ title, content });
```

Comment:
it's bad practice to push to the state if you want to edit it then you use setState and if you don't even use this data in the html then store it in an object